### DISCUSSION TITLE
blog/epbf-sidecar-and-service-mesh/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#284] [Announcements]
# blog/epbf-sidecar-and-service-mesh/

不管有没有 eBPF，在可预见的未来，服务网格都会基于运行在用户空间的 sidecar 代理（proxyless 模式除外）。

https://jimmysong.io/blog/epbf-sidecar-and-service-mesh/

**Comment by rootsongjc** [comment]
一个公司里搞eBPF的都是哪些团队？是不是安全的居多？
